File: Easy/Python/908. Smallest Range I.py
class Solution(object):
    def smallestRangeI(self, nums, k):
        nums.sort()
        if len(nums) == 1 or nums[len(nums) - 1] - k <= nums[0] + k:
            return 0
        
        return nums[len(nums) - 1] - nums[0] - k * 2

File: Easy/Java/908. Smallest Range I.java
class Solution {
    public int smallestRangeI(int[] nums, int k) {
        Arrays.sort(nums);
        if(nums.length == 1 || nums[nums.length - 1] - k <= nums[0] + k)
            return 0;
        
        return nums[nums.length - 1] - nums[0] - k * 2;
    }
}
